provisions/roles/cleanup/openshift/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Remove all containers
+  sudo: yes
+  shell: docker rm -f `docker ps -a -q`
+  ignore_errors: yes
+
+- name: Remove all images
+  sudo: yes
+  shell: docker rmi -f `docker images -a -q`
```

Comment:
yeah, if we remove all the images other than the openshift ones, that would suffice. @rtnpro we are cleaning up all the images to make sure we are not keeping the images created while building images. Ideally, there should not be any if pods are deleted properly while building. we can also plan for deleting the project to make sure this is taken care it self. but for now, lets remove all the images (or images except openshift ones if we can) as provision will bring everything back.